PFR/ForMathlib/Entropy/RuzsaDist.lean

```diff
@@ -177,19 +183,19 @@ lemma rdist_of_inj  {H:Type*} [hH : MeasurableSpace H] [MeasurableSingletonClass
       have : φ ∘ (fun x ↦ x.1 - x.2) = g ∘ f := by
         ext x
         simp
-      rw [this, ← MeasureTheory.Measure.map_map (g := g) (f := f) (measurable_of_countable _) (measurable_of_countable _), ← MeasureTheory.Measure.map_map (measurable_of_countable _) hX, ← MeasureTheory.Measure.map_map (measurable_of_countable _) hY]
+      rw [this, ← MeasureTheory.Measure.map_map (g := g) (f := f) (measurable_of_countable _)
+        (measurable_of_countable _), ← MeasureTheory.Measure.map_map (measurable_of_countable _) hX,
+        ← MeasureTheory.Measure.map_map (measurable_of_countable _) hY]
       congr
-      convert MeasureTheory.Measure.map_prod_map _ _ (measurable_of_countable _) (measurable_of_countable _)
+      convert Measure.map_prod_map _ _ Measurable.of_countable Measurable.of_countable
```

Comment:
Using the new "anonymous dot notation",
```suggestion
      convert Measure.map_prod_map _ _ .of_countable .of_countable
```
This also makes sure you put `Measurable.of_countable` in the right namespace

Reply:
I'm not at all sure how this is supposed to work exactly. What's the right namespace in `PFR.Mathlib.MeasureTheory.MeasurableSpace.Basic`? 

I've tried but I can't make it work. I'm sorry.

Reply:
https://leanprover-community.github.io/mathlib4_docs/Mathlib/MeasureTheory/MeasurableSpace/Defs.html#Measurable shows it's in the root namespace, so `Measurable.of_countable` should also be in the root namespace

Reply:
Isn't it already in the root namespace? No specific namespace is declared in this file https://github.com/pitmonticone/Polynomial_Freiman-Ruzsa_Conjecture/blob/e6d5c0ce3546128beeafefd356777a5a110ba104/PFR/Mathlib/MeasureTheory/MeasurableSpace/Basic.lean#L24-L26